src/Day05.kt

```diff
@@ -17,13 +17,12 @@ fun main() {
         var totalNiceList = 0
 
         for(string in input) {
-            when(isNice(string)) {
-                true        -> totalNiceList++
-                false       -> continue
-            }
+            if (isNice(string)) totalNiceList++
```

Comment:
This is a bit cleaner than your version with the `when`. Two things to keep in mind:
1. If you ever find yourself writing a `when` with only two branches then idiomatic Kotlin says you should write an `if else` instead.
2. If one of those branches is a `continue` or `break` or `return` then guaranteed you can simplify and just ignore one branch. In this case, the `continue` isn't really needed.

Reply:
Also note: because I changed the `return` below the code in the `for` loop here isn't really making any difference to the final output. Just a warning in case you are playing around with this and the output isn't doing what you expect!